### DISCUSSION TITLE
Welcome to BiliBiliToolPro Discussions!

### DISCUSSION THREAD

**Discussion by RayWangQvQ** [RayWangQvQ/BiliBiliToolPro#91] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gcorance** [comment]
企业微信推送现在需要增加信任ip，能否增加一个代理的环境变量选项？
（青龙面板）
